src/block-editor/format-types/inline-binding/components/InlineBindingSelectFieldPopover.tsx

```diff
@@ -28,6 +29,15 @@ export function InlineBindingSelectFieldPopover( props: InlineBindingSelectField
 	} );
 	const { remoteData, selectedField, type } = props.fieldSelection;
 
+	// For now, we will use the first compatible selector, but this should be improved.
+	// Same as InlineBindingSelection.tsx
+	const compatibleSelector = getBlockConfig( remoteData?.blockName ?? '' )?.selectors.find(
```

Comment:
I've kept this just like it is when the selector is first picked out for the inline binding selection. I do want to fix this, but I also want to get this working end to end. Once the query format has been finalized, I'll see if I can tweak/fix this.